app/controllers/notifications_controller.rb

```diff
@@ -0,0 +1,11 @@
+class NotificationsController < ApplicationController
+  def index
+    @user = current_user
```

Comment:
It looks like `@user` is not used in views. You can do `@notifications = current_user.notifications.order('created_at DESC')` then